storage/integration_test/src/integration_test.cc

```diff
@@ -588,7 +588,10 @@ TEST_F(FirebaseStorageTest, TestLargeFilePauseResumeAndDownloadCancel) {
     // operation.
     ASSERT_TRUE(controller.is_valid());
 
-    ProcessEvents(500);
+    while(controller.bytes_transferred() == 0) 
+    {
+      ProcessEvents(1);
+    }
     // After waiting a moment for the operation to start, pause the operation
     // and verify it was successfully paused. Note that pause might not take
     // effect immediately, so we give it a few moments to pause before
```

Comment:
Do the comments need to be updated to reflect the new strategy?

Reply:
Nope, this is the same behavior as before.

Reply:
hmm I guess we're just being smarter about how we're "waiting a moment". Fair.

Reply:
Though I don't think that comment was accurate even in the previous implementation.  updating it.